internal/db/schema/migrations/postgres/14/01_wh_user_dimension_oidc.up.sql

```diff
@@ -0,0 +1,154 @@
+begin;
+
+  alter table wh_user_dimension
+    add column auth_method_external_id  wh_dim_text,
+    add column auth_account_external_id wh_dim_text,
+    add column auth_account_full_name   wh_dim_text,
+    add column auth_account_email       wh_dim_text
+  ;
+
+  drop view whx_user_dimension_source;
+  create view whx_user_dimension_source as
+       select -- id is the first column in the target view
+              u.public_id                       as user_id,
+              coalesce(u.name, 'None')          as user_name,
+              coalesce(u.description, 'None')   as user_description,
+              coalesce(aa.public_id, 'None')    as auth_account_id,
+              case
+                   when apa.public_id is not null then 'password auth account'
+                   when aoa.public_id is not null then 'oidc auth account'
+                   else 'None'
+                   end                          as auth_account_type,
+              coalesce(apa.name, 'None')        as auth_account_name,
+              coalesce(apa.description, 'None') as auth_account_description,
```

Comment:
I think the `auth_oidc_account` table also has name and description columns. If so, these need to be `case` statements.